css/style.css

```diff
@@ -0,0 +1,609 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Staatliches&display=swap');
+
+:root {
+  --color-primary: #575757;
+  --color-dark: #272a31;
+  --color-accent: #ec5242;
+  --color-negative: #d3d3d3;
+  --color-white: #fff;
+  --ff-text: 'Lato', sans-serif;
+  --ff-title: 'COCOGOOSE', sans-serif;
+  --max-width: 75vw;
+}
+
+/* body * {
+  outline: 1px solid blue ;
+} */
```

Comment:
- [ ] Kindly remove the inline comments from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1: